src/styles/header.scss

```diff
@@ -0,0 +1,84 @@
+@import './variables';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  max-width: 100%;
+  min-width: 1024px;
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  padding: 0 50px;
+  box-sizing: border-box;
+
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: #{$blue-accent};
```

Comment:
It's better to use variables for all colors in your code. This will make it easier to maintain and update the color scheme of your project.